lk2nd/smp/clock-msm8953.c

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/* Copyright (c) 2022, Stephan Gerhold <stephan@gerhold.net> */
```

Comment:
@stephan-gh is this fine? it is basing from clock-msm8916.c what was created by you so guess i should keep the copyright header from it or idk.